pkg/workspace/manifests/manifests.go

```diff
@@ -98,7 +98,7 @@ func GenerateServiceManifest(workspaceObj *kaitov1beta1.Workspace, serviceType c
 func GenerateStatefulSetManifest(workspaceObj *kaitov1beta1.Workspace, revisionNum string, imageName string,
```

Comment:
This feature only supports StatefulSets for now?

Reply:
Yes. Because it's local pv, cannot be shared across nodes. 
If I add this to deployment, i think it would fail when scaling that deployment.

I am considering migrating all of them to statefulset.  WDTY